test/util/fake-xml-http-request-test.js

```diff
@@ -1710,6 +1710,20 @@ if (typeof window !== "undefined") {
                 this.xhr.respond(200, {}, "");
             });
 
+            it("calls #onload on success for 403", function (done) {
```

Comment:
Make the message more generic. This is not about 403 specifically. It's about the non-200 requests.